### ISSUE TITLE
Add Wow Community APIs

### ISSUE THREAD

**Issue by avivbiton** [avivbiton/BlizzardApiReader#3]
The APis docs can be found here https://dev.battle.net/io-docs

**Comment by redmars27** [comment]
Have you think about how to handle action house data? I might be able to contribute.

**Comment by avivbiton** [comment]
Not really, I didn't take an in-depth look into the WoW APIs since I am unable to test them anyway.  
Does handling action house data require anything special we don't already handle? feel free to contribute and post a pull request when you're done. Thank you.

**Comment by redmars27** [comment]
It seems you need to do two calls, one for checking the latest dump generated from the auction house and another one to actually download the dump. I am still learning how to work in open source way, but expect some pull requests from my side. Let me ask for forgiveness in advance if I do something wrong.

**Comment by avivbiton** [comment]
Alright, if you have any issues, feel free to ask.
